### PR TITLE
Update VS Code debugging setting and instructions

### PR DESCRIPTION
**Pull request by troydai**

### CODE REVIEW

File: doc/debug/debug_in_vs_code.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Debug in Visual Studio Code
+
+## Prerequisit
```
Comment (derekbekoe) [inline doc/debug/debug_in_vs_code.md]: Typo 'Prerequisite'?

Comment (troydai) [inline doc/debug/debug_in_vs_code.md]: Yup! Will fix.

**Comment by troydai** [comment]
Add fix. Will wait for CI (hopefully not another 11 hours) and then squash and merge.

**Comment by troydai** [comment]
Thanks, @tjprescott @derekbekoe .

**Pull request merged**